system/wymeditor/plugins/image_upload/jquery.wymeditor.image_upload.js

```diff
@@ -40,9 +40,14 @@ WYMeditor.editor.prototype.image_upload = function() {
 			},
 			complete: function(response){
 				response = response[0];
-				jQuery(".wym_src", doc).val(response.thumbUrl);
-				jQuery(".wym_alt", doc).val(response.original_filename);
-				jQuery("form#image_upload_form .submit", doc).val(oldSubmitLabel);
+				if(response.error){
+                    alert(response.error.substring(1));
+                    document.getElementById('upload-image').value = oldSubmitLabel;
+                }else {
+                    jQuery(".wym_src", doc).val(response.thumbUrl);
+                    jQuery(".wym_alt", doc).val(response.original_filename);
+                    jQuery("form#image_upload_form .submit", doc).val(oldSubmitLabel);
+                }
```

Comment:
jQuery("form#image_upload_form .submit", doc).val(oldSubmitLabel);

Reply:
Looking at the file contents, the original file uses tabs for indentation, but your inserted lines use 4 spaces. Might be good to fix in your editor.

Reply:
Fixed.